README.org

```diff
@@ -21,6 +21,9 @@ These attributes are named =emacsGit= and =emacsUnstable=.
 Emacs from git is not guaranteed stable and may break your setup at any
 time, if it breaks you get to keep both pieces.
 
+The =feature/native-comp= branch is also provided under the attribute =emacsGcc=.
```

Comment:
Should call it `gccemacs` to match the project name?

Reply:
I thought about it, but I wanted to be consistent with all other attributes that start with `emacs...` .